cms/utils/__init__.py

```diff
@@ -16,15 +21,42 @@ def get_current_site():
     return Site.objects.get_current()
 
 
+@lru_cache(512)
```

Comment:
What is the justification for the value? Also lru_cache is not Python 2.7 compat is it?

Reply:
You are correct. Python 2.7 is end of life though. Is django-cms committed to maintain this for 2.7? For how long?

Reply:
in travis.yml they allow failures for python 2.7. So it looks like it is as good as deprecated.

Reply:
I agree that it is as good as dead, issue is the project still supports it so our implementation should keep that support until a decision to not support is broadcast. We don't have that authority and It's changes like this that could jeopardise an agreement from Divio to support our changes. 2.7 is still in develop as supported: https://github.com/divio/django-cms/blob/develop/setup.py

Reply:
Our target is to merge into develop soon.